src/yb/consensus/retryable_requests.cc

```diff
@@ -244,18 +244,37 @@ std::ostream& operator<<(std::ostream& out, const ReplicateData& data) {
 
 } // namespace
 
+std::string LogPrefix(const std::string& tablet_id, const std::string& uuid) {
+  return "T " + tablet_id + " P " + uuid + ": ";
+}
+
+RetryableRequestsManager::RetryableRequestsManager(
+    const TabletId& tablet_id, FsManager* const fs_manager, const std::string& wal_dir,
+    const MemTrackerPtr& mem_tracker, const std::string log_prefix)
+    : tablet_id_(tablet_id),
+      fs_manager_(fs_manager),
+      dir_(wal_dir),
+      retryable_requests_(std::make_unique<RetryableRequests>(mem_tracker, std::move(log_prefix))),
+      log_prefix_(std::move(log_prefix)) {
+    // Several tests (e.g. tablet_peer-test) erroneously set the log_prefix argument to ""
```

Comment:
I think we can explicitly set the log_prefix from upper layer, otherwise, leave it as empty.